app/uploaders/picture_uploader.rb

```diff
@@ -52,6 +52,18 @@ def store_dir
     "system/#{object_class_name}/#{mounted_as}/#{model.id}"
   end
 
+  def image
+    @image ||= MiniMagick::Image.open( file.file )
```

Comment:
Rubocop doesn't like this... :sob: 
Try to remember to run Rubocop locally before pushing :wink: